angrmanagement/__main__.py

```diff
@@ -97,53 +38,16 @@ def set_windows_event_loop_policy():
         asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
 
 
-def macos_bigsur_wants_layer():
-    # workaround for https://bugreports.qt.io/browse/QTBUG-87014
-    # this is because the latest PySide6 (5.15.2) does not include this fix
-    v, _, _ = platform.mac_ver()
-    vs = v.split(".")
-    if len(vs) >= 2:
-        major, minor = [int(x) for x in vs[:2]]
-    else:
-        return
-    if major >= 11 or major == 10 and minor == 16:
-        os.environ['QT_MAC_WANTS_LAYER'] = '1'
```

Comment:
Can you confirm this QT_MAC_WANTS_LAYER config is no longer needed

Reply:
The linked bug report suggests that it is fixed in newer versions of qt5 and qt6, and the pyinstaller bundle seems to start just fine on macos 12 with this patch.